packages/core/contracts/financial-templates/common/FundingRateApplier.sol

```diff
@@ -80,28 +80,28 @@ abstract contract FundingRateApplier is EmergencyShutdownable, FeePayer {
     }
 
     // Note: this modifier is intended to be used if the caller intends to _only_ pay regular fees.
-    modifier regularFees {
+    modifier paysRegularFees {
```

Comment:
I think we should rename this modifier in `FeePayer` as well

Reply:
Looks like the `fees` modifier in `FeePayer` gets overriden in `FundingRateApplier`, so this would need a different name than `paysRegularFees` if we're changing it.